velox/docs/monthly-updates/december-2022.rst

```diff
@@ -0,0 +1,37 @@
+********************
+December 2022 Update
+********************
+
+Presto Functions
+================
+
+* Add :func:`array_has_duplicates`, :func:`crc32`, :func:`hmac_sha1`,
+  :func:`hmac_sha256`, :func:`hmac_sha512`, :func:`json_size` scalar functions.
+
+Spark Functions
+===============
+
+* Add shiftleft and shiftright scalar functions.
+
+Substrait
+=========
+
+* Add support for OrderBy/TopN/Limit plan nodes.
+
+Performance and Correctness
+===========================
+
+* Add nightly job to run Expression Fuzzer using Spark SQL functions.
+* Add nightly job to run Join Fuzzer.
+* Add basic support for window queries to Aggregation Fuzzer.
+* Add support for CAST and SWITCH special forms to Expression Fuzzer.
+
```

Comment:
Can we add a seperate section for build system improvements ? 

```
Build System Improvements
======================

* Moved Boost from pre installed system dependency to being automatically installed via CMake resolve_dependency. 
* Nightly docker images for presto_native against latest Velox
* Nightly docker build jobs against ubuntu and centos OS's
```

Reply:
Also not sure if this should be separate section, but can we also add: 
```
 * PyVelox python package with python bindings for the following velox api's:
   - Conversions from python lists to Velox flat vectors
   - Support for construction of expressions from strings and their evaluation

```

Reply:
@vibhata is working on the documentation, theres a PR out which should be ready to land soon.